### DISCUSSION TITLE
Data and GIS can be intentionally manipulated to fulfill a nefarious agenda.

### DISCUSSION THREAD

**Discussion by lvalles10** [albertkun/21S-ASIAAM-191A#41] [Past Thinking Caps]
Data and GIS can be intentionally manipulated to misinform readers or achieve a nefarious agenda. For example, Figures 0.1 and 0.2 in Hands on Data Viz both depict the Average US Adult Income, by Percentile, 1970-2019 but exhibit vastly different realities of income inequity. This is because Figure 0.2 follows a logarithmic scale, which reduces the intensity of the income inequity displayed in the graph. In this way, GIS is a vehicle for misinformation and deflation of important civil realities. 

Furthermore, as mentioned in Discussion earlier this week, Civil Society and Web Development can function as an effective method of change when GIS, Coding, and Ethics all interact in a manner conscious of broader social issues. But, at the same time, when GIS is utilized without critical cartography or data studies (or even worse in blatant disregard of these theories) it can propagate misinformation. That's why it's imperative for Ethics to interact with GIS and Coding; I'm excited to exercise this critical form of thinking throughout the course!

**Comment by epham36** [comment]
Definitely! Data visualization is such an important tool to make data more digestible and accessible through the use of visual aids. However, when it is manipulated to deceive our interpretation of data at first sight, it can become a dangerous tool, which can skew interpretations of data, changing perspectives, beliefs, and actions. 

Agreed! I think it's so great to be working with the ethics of coding and geographical data! Given how inaccessible data can be and how relatively few people have the skill sets to create data and maps (whether they are intentionally manipulative or not), it's so important to have these skills to critique cartography and to have the agency to create our own depictions of data and geography.

**Comment by latishasumardy** [comment]
I agree. Especially in this day and age, data manipulation through misinformation is becoming very prevalent, and it is important for us as consumers to learn how data is being misused so that we can spot it when consuming media. By learning how to understand the maps and read them correctly, we can slow the spread of misinformation and biased data. I also think that it is important to understand the technical detail behind how the maps are created, to further understand what data is being manipulated and how.
